docs/development.md

```diff
@@ -300,50 +300,38 @@ you need more samples and therefore a longer running test. This is related to th
 [This tool](http://powerandsamplesize.com/Calculators/Compare-2-Proportions/2-Sample-Equality) can help to 
 compute the sample size of your experiment, from the sample size, you can then estimate the duration of your test.  
 
-#### Step 1: Add your test to the Tests array
+#### Step 1: Add your test to the Tests object
 
  After your hypothesis is defined, you can implement the test in the codebase. First, define the test 
- in the [`abtest.js` shared helper](/assets/helpers/abtest.js) by adding a new object to the tests array (under the **Tests** section). 
+ in the [`abtest.js` shared helper](/assets/helpers/abtests.js) by adding a new object to the tests array (under the **Tests** section). 
  
  ```javascript 1.8
  // ----- Tests ----- //
  
- const tests: Test[] = [
-   {
-     testId: 'yourTestId', 
-     variants: ['control', 'variantA'],  
+ export type Tests = { [testId: string]: Test }
+ 
+ export const tests: Tests = {
+   yourTestId: {
+     variants: ['control', 'variant'],
      audiences: {
-       GB :{
-          offset: 0,
-          size: 1,
-        }
+       GB: {
+         offset: 0,
+         size: 1,
+       },
      },
      isActive: true,
```

Comment:
I know this is probably not related to this PR, but can you add the `independence: 1,` line to this example. In that way the docs would be up to date.